specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-04-01/cosmos-db.json

```diff
@@ -5281,13 +5281,19 @@
           "readOnly": true
         },
         "_ts": {
+          "type": "number",
```

Comment:
Can you please change the description in line 5275. That is outdated, we support these properties for MongoDB, Cassandra and Table resources as well. 
Second, let us not merge this until the corresponding change in RP is checked in and deployed in Prod.

Reply:
Do I change the description to "The system generated resource properties" then?